cmake/Modules/CMakePolicy.cmake

```diff
@@ -10,3 +10,9 @@ endif()
 if(POLICY CMP0116)
   cmake_policy(SET CMP0116 OLD)
 endif()
+
++# Enable debug info compatible with SCCache
```

Comment:
This explains the end goal of why we're adding this policy here, at this time, but it doesn't say anything to the reader about what `CMP0141` really does. (Yes, the reader can google it, but it's good with a quick reference here inline.) Can you do a 1-2 line brief summary of what it changes, like for the other policies above (and mention in which CMake version it was introduced)? It's fine to keep the end motivation for why we're adding it though.

Reply:
OK, following the precedent set by the other policies, I'll add CMake's own summary: [MSVC debug information format flags are selected by an abstraction](https://cmake.org/cmake/help/latest/policy/CMP0141.html).  I find it barely comprehensible, and don't find it very useful.  A more accessible explanation is that without the flag, `cl` complains of what looks like multiple simultaneous writes to the same .pdb file, so maybe I'll add that too.